db/write_thread.h

```diff
@@ -83,6 +83,8 @@ class WriteThread {
     std::atomic<size_t> running;
     size_t size = 0;
 
+    ~WriteGroup() { status.PermitUncheckedError(); }
```

Comment:
Can you add a comment about why this can be permitted?

Reply:
Actually, I think it's better not to call `PermitUncheckedError()` in destructor. Let me update the code.